crates/core/tedge_agent/src/main.rs

```diff
@@ -29,12 +30,6 @@ pub struct AgentOpt {
     #[clap(short, long)]
     pub init: bool,
 
-    /// Start the agent with clean session on, drop the previous session and subscriptions
-    ///
-    /// WARNING: All pending messages will be lost.
-    #[clap(short, long)]
-    pub clear: bool,
```

Comment:
A bit out of scope. But it's okay as this is was not a good idea to propose this option.